pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/Utils.groovy

```diff
@@ -168,6 +168,25 @@ class Utils {
         }
     }
 
+    // TODO: Is there some magic trick for filtering out *nested* stages from a list? If so, that'd be handy.
+    /**
+     * This will return true for flow nodes in *child* stages, not just the immediate enclosing stage.
+     *
+     * @param stageStartNode
+     * @return A predicate that returns true if the applied input is somewhere within the given stage
+     */
+    static Predicate<FlowNode> isSomewhereWithinGivenStage(final FlowNode stageStartNode) {
```

Comment:
```suggestion
    static Predicate<FlowNode> isSomewhereWithinStage(final FlowNode stageStartNode) {
```
Sorry, it's the "Given" that's bothering me.

Reply:
heheh, ok.